src/main/java/hudson/tasks/SMTPAuthentication.java

```diff
@@ -0,0 +1,40 @@
+package hudson.tasks;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import hudson.util.Secret;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+public class SMTPAuthentication extends AbstractDescribableImpl<SMTPAuthentication> {
+
+    private String smtpAuthUsername;
+
+    private Secret smtpAuthPassword;
+
+    @DataBoundConstructor
+    public SMTPAuthentication(String smtpAuthUsername, Secret smtpAuthPassword) {
```

Comment:
The `smtpAuth` prefix on these property names is redundant. Better to switch to just `username` & `password`.